app/src/main/java/com/cornellappdev/scoop/ui/components/general/DatePicker.kt

```diff
@@ -0,0 +1,110 @@
+package com.cornellappdev.scoop.ui.components.general
+
+import android.util.Log
+import androidx.compose.foundation.border
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material.*
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.MutableState
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.vector.ImageVector
+import androidx.compose.ui.platform.LocalContext
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.TextStyle
+import androidx.compose.ui.text.style.TextOverflow
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import com.cornellappdev.scoop.R
+import com.cornellappdev.scoop.ui.components.post.createDatePickerDialog
+import com.cornellappdev.scoop.ui.theme.PlaceholderGray
+import java.text.SimpleDateFormat
+import java.util.*
+
+/**
+ * Allows users to select a city using Place API autocomplete.
```

Comment:
If this is what this composable should be doing, why is it called DatePicker? I'm assuming this comment is wrong as I haven't read the rest of the code yet. So fix it please

Reply:
LMFAo mb